json_serializable/tool/readme/readme_template.md

```diff
@@ -1,23 +1,22 @@
 [![Pub Package](https://img.shields.io/pub/v/json_serializable.svg)](https://pub.dev/packages/json_serializable)
 
-Provides [Dart Build System] builders for handling JSON.
```

Comment:
I generally like cleanup, but please don't mix unrelated things in this PR. Makes it hard to review!

Reply:
I don't know what happened, I didn't make these changes myself. I reverted them.